Header header  

uint8 NOTHING=0         # Do nothing
uint8 WAIT=1            # Wait for a different action to execute
uint8 MAINTAIN=2        # Maintain engagement with a target
uint8 RECAPTURE=3       # Recapture engagement with a target
uint8 ELICIT_TARGET=4   # Elicit engagement with a target
uint8 ELICIT_GENERAL=5  # Elicit engagement from anyone in general

uint8 decision
string target